ksql-engine/src/test/java/io/confluent/ksql/function/udf/json/ArrayContainsKudfTest.java

```diff
@@ -0,0 +1,144 @@
+package io.confluent.ksql.function.udf.json;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class ArrayContainsKudfTest
+        extends TestCase
+{
+    private ArrayContainsKudf jsonUdf = new ArrayContainsKudf();
+
+    @Override
+    public void setUp()
+            throws Exception
+    {
+        jsonUdf.init();
+    }
+
+    @Test
+    public void testEmptyArray()
```

Comment:
i prefer test names that tell me what it is the object should do, i.e, `shouldReturnFalseOnEmptyArray`